### DISCUSSION TITLE
Documentation site not available, it returns ERR_NAME_NOT_RESOLVED

### DISCUSSION THREAD

**Discussion by klinux** [terrakube-io/terrakube#412] [Q&A] [answered]
I'm trying to access the docs and terrakube site, but I'm receiving ERR_NAME_NOT_RESOLVED

There's some problems with docs and portal?

**Comment by alfespa17** [comment]
Hello @klinux 

We are having some issues with the site, we hope to fix that today

**Comment by jcanizalez** [comment]
Hi @klinux 

Please use https://docs.terrakube.io/

**Reply by klinux** [reply, marked as answer]
it's works, thank you
